### ISSUE TITLE
Website basic configuration

### ISSUE THREAD

**Issue by riofriz** [arventurers/shared-testing#8]
Was looking at that app this morning (the drawing wherever you are with other people seeing your drawing in ar) and i was thinking, after somebody scans their first image, if they are not logged in into the website they could create an account and earn badges and reminders.

Say we do the story mode, they could be able to see where they are at, what they're missing, easter eggs and stuff like that.. Create a portal of some sort.

I think it could be a cool interactive experience.

**Comment by twanito** [comment]
Agreed that's next level implementation. Do you think it'd be difficult or are there thousands of existing frameworks/templates to do so?
